signature/src/schnorr.rs

```diff
@@ -597,19 +672,14 @@ mod tests {
                 let msg = vec![$base_field::rand(&mut rng)];
                 let sig = keypair.sign(&msg, CS_ID_SCHNORR);
 
-                let mut ser_bytes: Vec<u8> = Vec::new();
-                keypair.serialize_compressed(&mut ser_bytes).unwrap();
-                let de: KeyPair<$curve_param> =
-                    KeyPair::deserialize_compressed(&ser_bytes[..]).unwrap();
-                assert_eq!(de.ver_key_ref(), keypair.ver_key_ref());
-                assert_eq!(de.ver_key_ref(), &VerKey::from(&de.sk));
-
-                let mut ser_bytes: Vec<u8> = Vec::new();
-                sk.serialize_compressed(&mut ser_bytes).unwrap();
-                let de: SignKey<$scalar_field> =
-                    SignKey::deserialize_compressed(&ser_bytes[..]).unwrap();
+                let mut ser_bytes: Vec<u8> = sk.to_bytes();
+                let de = SignKey::<$scalar_field>::from_bytes(&ser_bytes);
```

Comment:
this is good, but let's also add a completeness test for `TryFrom<TaggedBase64>` and `to_tagged_base64()`

Reply:
done in [5f85f10](https://github.com/EspressoSystems/jellyfish/pull/696/commits/5f85f1048212d738738a5884784269f1aea54d71)